react-hs-examples/redux/stack.yaml

```diff
@@ -0,0 +1,11 @@
+resolver: lts-8.3
+packages:
+- .
+- location:
+    git: https://github.com/liqula/react-hs
+    commit: e6260f0162f1ced6dc03e54e2f1e24825369949a
+  extra-dep: true
+  subdirs:
+  - react-hs
```

Comment:
The reason I went for this approach is so that we can have lots of examples without making the library too hard to maintain. As currently any breaking change would involve having to go through every example and fix them all, which is particularly hard since each example will probably take its own approach and have its own style, making it difficult for one person to fix it all.

Particularly since once react-hs is on hackage then it would make much more sense to have the examples be based on actual published versions of react-hs that people are using rather than on the current development branch.

I would say that once react-hs is on hackage it makes sense to have some sort of "examples cannot be more than X versions behind or they must be promptly updated or removed" policy. But for now even just a "examples cannot be more than X commits behind" might not be the worst, it would be best to just get react-hs on hackage ASAP though.